src/ArduboyCore.cpp

```diff
@@ -242,54 +242,33 @@ void ArduboyCore::paintScreen(const uint8_t *image)
 
 // paint from a memory buffer, this should be FAST as it's likely what
 // will be used by any buffer based subclass.
-// if this function is changed, make sure corresponding changes
-// are made to paintScreenAndClearImage()
-void ArduboyCore::paintScreen(uint8_t image[])
+void ArduboyCore::paintScreen(uint8_t image[], bool clear)
```

Comment:
A default of _false_ **is** set in the function prototype in _ArduboyCore.h_. You set default values in the prototype, not the definition.